lib/matplotlib/artist.py

```diff
@@ -446,8 +446,8 @@ def pick(self, mouseevent):
         for a in self.get_children():
             # make sure the event happened in the same axes
             ax = getattr(a, 'axes', None)
-            if mouseevent.inaxes is None or ax is None or \
-                    mouseevent.inaxes == ax:
+            if (mouseevent.inaxes is None or ax is None
+                    or mouseevent.inaxes == ax):
```

Comment:
I tend to (very much) favor the newer PEP8 recommendations (https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator) which I think is usually more readable.